src/aap_eda/api/serializers/activation.py

```diff
@@ -585,6 +586,10 @@ class ActivationReadSerializer(serializers.ModelSerializer):
     organization = OrganizationRefSerializer()
     rules_count = serializers.IntegerField()
     rules_fired_count = serializers.IntegerField()
+    ruleset_stats = YAMLSerializerField(
```

Comment:
Can you explain to me what this would look like? It's stored as yaml and then serializes it based on the requested Content-Type?

Reply:
@matburt Yes, the added `ruleset_stats` output looks like this when the rulebook has two rulesets named `set1` and `set2`:
```
{
  "id": 7,
  "name": "ccc",
  "description": "",
  "is_enabled": true,
......
  "restart_policy": "never",
  "restart_count": 0,
  "rulebook_name": "multiple_rulesets.yml",
  "current_job_id": null,
  "ruleset_stats": {
    "set1": {
      "end": "2024-08-21T18:39:52.441281699Z",
      "start": "2024-08-21T18:38:52.019259901Z",
      "sessionId": 1,
      "usedMemory": 5534120,
      "ruleSetName": "set1",
      "eventsMatched": 2,
      "lastClockTime": "2024-08-21T18:39:52.419Z",
      "lastRuleFired": "rule2",
      "numberOfRules": 2,
      "asyncResponses": 0,
      "rulesTriggered": 2,
      "eventsProcessed": 6,
      "lastRuleFiredAt": "2024-08-21T18:39:12.319Z",
      "bytesSentOnAsync": 0,
      "eventsSuppressed": 4,
      "clockAdvanceCount": 604,
      "maxAvailableMemory": 518979584,
      "lastEventReceivedAt": "2024-08-21T18:39:42.319Z",
      "permanentStorageSize": 0,
      "numberOfDisabledRules": 0,
      "permanentStorageCount": 0
    },
    "set2": {
      "end": "2024-08-21T18:39:52.487413029Z",
      "start": "2024-08-21T18:38:52.031966388Z",
      "sessionId": 2,
      "usedMemory": 5549408,
      "ruleSetName": "set2",
      "eventsMatched": 2,
      "lastClockTime": "2024-08-21T18:39:52.431Z",
      "lastRuleFired": "rule4",
      "numberOfRules": 2,
      "asyncResponses": 0,
      "rulesTriggered": 2,
      "eventsProcessed": 6,
      "lastRuleFiredAt": "2024-08-21T18:39:12.331Z",
      "bytesSentOnAsync": 0,
      "eventsSuppressed": 4,
      "clockAdvanceCount": 604,
      "maxAvailableMemory": 518979584,
      "lastEventReceivedAt": "2024-08-21T18:39:42.431Z",
      "permanentStorageSize": 0,
      "numberOfDisabledRules": 0,
      "permanentStorageCount": 0
    }
  },
  "rules_count": 4,
  "rules_fired_count": 4,
  "created_at": "2024-08-21T18:38:39.635561Z",
  "modified_at": "2024-08-21T18:40:00.637544Z",
  "restarted_at": null,
  "status_message": "Pod e9765465b60e35a2b7468cb45918f68f4f55853cba4ba4893c82c168be6a3d83 has successfully exited. Activation completed successfully. No restart policy is applied.",
......
  "skip_audit_events": false
}
```

Reply:
@matburt are you ok with this output?